inst/templates/Makevars.in

```diff
@@ -1,9 +1,11 @@
 TARGET_DIR = ./rust/target
-LIBDIR = $(TARGET_DIR)/release
+LIBDIR = $(TARGET_DIR)/@LIBDIR@
 STATLIB = $(LIBDIR)/lib{{{lib_name}}}.a
 PKG_LIBS = -L$(LIBDIR) -l{{{lib_name}}}
 
-all: C_clean
+all: $(SHLIB) rust_clean
```

Comment:
used to clean up target and vendor directory which is the main resolution for the new WARNING